doc/nrf/protocols/thread/certification.rst

```diff
@@ -177,3 +180,86 @@ This is valid only for Nordic Semiconductor development kits with a J-Link virtu
 
 To add an nRF52840 DK, drag the nRF52840 DK and drop it on the test bed configuration page.
 After that, the device is configured and the :ref:`proper baud rate (115200) <test_and_optimize>` and COM port are set.
+
+.. _ug_thread_build_report:
+
+OpenThread build report
+***********************
+
+The OpenThread build report contains information about:
+
+   * The current |NCS| and OpenThread revisions.
+   * The Thread feature set and Thread library path.
+   * Changes in the :ref:`nrfxlib:nrfxlib` repository in comparison to the latest |NCS| release.
+
+The report is generated to the output console log, and stored as an additional build artefact in the application build directory.
+
+Generating the OpenThread report is enabled by default if the :kconfig:option:`CONFIG_NET_L2_OPENTHREAD` is set to ``y``.
+It means that it is enabled for all samples that use Thread stack.
+To disable the generation, set the :kconfig:option:`CONFIG_OPENTHREAD_REPORT` kconfig option to ``n``.
+You can disable generating the build artefact by setting the :kconfig:option:`CONFIG_OPENTHREAD_REPORT_BUILD_ARTEFACT` kconfig option to ``n``.
+
+By default, the build artefact name is set as :file:`ot_report.txt`, but you can specify a different name by setting the :kconfig:option:`CONFIG_OPENTHREAD_REPORT_BUILD_ARTEFACT_NAME` kconfig value to the new one.
+
+Depending on the way of delivering OpenThread libraries you will see the following logs in your build console:
+
+.. tabs::
+
+   .. group-tab:: building the application using the :ref:`nrfxlib:ot_libs`
```

Comment:
The tab names seem to be a bit too long:
![image](https://github.com/user-attachments/assets/5c937881-0b2a-42b2-ac4e-5cc30335c972)
So moving some of the information to the introducing sentence.
```suggestion
Depending on if you build the application using the :ref:`nrfxlib:ot_libs` or if you build the application and Thread stack from the source files, you will see the following logs in your build console:

.. tabs::

   .. group-tab:: Using pre-built libraries
```